tests/matrix_free/interpolate_rt.output

```diff
@@ -0,0 +1,99 @@
+
+DEAL:2d::Testing FE_RaviartThomasNodal<2>(0)
+DEAL:2d::Error cell   values:     0.77942 0.31754 
+DEAL:2d::Error cell   gradients:  2.7000 1.1000 
+DEAL:2d::Error cell   divergence: 2.2204e-16 
```

Comment:
Note that it is expected that RT with tangential degree 0 (normal degree 1) can neither represent the value or gradient of a linear function exactly; however, the divergence is exactly represented because the normal direction can capture the linear function, which is all that matters for the divergence.